content/develop/concepts/multipage-apps/widgets.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Working with widgets in multipage apps
+slug: /develop/concepts/multipage-apps/widgets
+description: Understand how widgets interact with pages
+---
+
+# Working with widgets in multipage apps
+
+When you call a widget function in a Streamlit app, Streamlit creates a widget ID and uses it to make your widget stateful. As your users interact with your app and your app reruns, Streamlit keeps track of the widget's value by associating its value to its ID. In particular, a widget's ID depends on the page it's called from. If you define an identical widget on two different pages, then the widget will reset to its default value when you switch pages.
```

Comment:
1. Could we say `create a widget` instead of `call a widget function`? Thoughts? (the phrase is used a couple times on this page)
2. Could we link to the definition of "widget" in the first sentence? Or otherwise make it clearer / remind what we mean by this term?

Reply:
There is a usefulness to "call a widget function" since I can talk about both initializing a widget and the subsequent calls on reruns. I might be able to reword it here, but it's a little trickier later on when discussing disconnecting and reconnecting a widget to a particular key in Session State:
> ...Streamlit will not associate the key-value pair to any widget until you call a widget function again with that key.

Reply:
sounds good, fine to leave it how it is

Reply:
I would lean towards `create a widget`, too. We've always intentionally not called them functions. [Streamlit brand style guide](https://www.notion.so/snowflake-corp/Streamlit-brand-style-guide-b05db4418b44480ca50ba36e906bb3f9?pvs=4#0979867a1e5a41ca90a2499c7ac4df05)

Reply:
I can say "widget command" instead of "widget function" but I definitely need language to separate the command from the frontend component.